nautobot/virtualization/filters.py

```diff
@@ -182,18 +184,26 @@ class VirtualMachineFilterSet(NautobotFilterSet, LocalContextFilterSet, TenancyF
         method="_has_primary_ip",
         label="Has a primary IP",
     )
+    primary_ip = MultiValueCharFilter(
```

Comment:
Instead of adding `primary_ip` as a multi-field method filter, I thought we'd planned to just add distinct `primary_ip4` and `primary_ip6` filters?